### PR TITLE
Ignore plan if no access for creating treeItem

### PR DESCRIPTION
**Pull request by nturinski**
Related https://github.com/Microsoft/vscode-azureappservice/issues/769

Of course, this leads to more issues such as these: https://github.com/Microsoft/vscode-azureappservice/issues/946

### CODE REVIEW

File: src/explorer/WebAppTreeItem.ts [hunk 1]
```diff
@@ -30,8 +30,16 @@ export class WebAppTreeItem extends SiteTreeItem {
...
-        const asp: AppServicePlan | undefined = await this.root.client.getAppServicePlan();
-        const tier: string | undefined = asp && asp.sku && asp.sku.tier;
+        let tier: string | undefined
+        let asp: AppServicePlan | undefined;
+        try {
+            asp = await this.root.client.getAppServicePlan();
+            tier = asp && asp.sku && asp.sku.tier;
+        } catch (err) {
+            // ignore this error, we don't want to block users for deployment slots
+            tier = 'ignored';
```
Comment (EricJizbaMSFT) [inline src/explorer/WebAppTreeItem.ts]: I would call this 'unknown'

**Comment by nturinski** [comment]
Nah, this only fixes this specific error.

image

The other changes have to be made in the `deploy` command in the `appservice` package.

**Comment by nturinski** [comment]
Are we okay with this behavior?  This command is used very much and I think the error makes sense, but wanted your opinion @EricJizbaMSFT 

deployscript_failure

**Comment by EricJizbaMSFT** [comment]
Yeah that error is perfectly reasonable IMO

**Pull request merged**